src/main.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
 import { HelmetProvider } from 'react-helmet-async';
-import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { createBrowserRouter, RouterProvider } from 'react-router';
 import { Metric } from 'web-vitals';
 import { ErrorNoteProvider } from './components/ErrorNote';
 import './i18n/config';
 import './index.css';
 import { routes } from './routes';
 
-const router = createBrowserRouter(routes, { basename: '/yksilo' });
+const router = createBrowserRouter(routes, {
+  basename: '/yksilo',
+  future: {
+    v7_relativeSplatPath: true,
```

Comment:
Is this a leftover? I think these future settings are just for trying out upcoming changes before migration.